ICS372-A1/src/DealerGroup.java

```diff
@@ -0,0 +1,75 @@
+
+
+import java.util.ArrayList;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.Random;
+
+import org.json.*;
+
+public class DealerGroup {
+
+	private ArrayList<Dealership> dealers;
+	
+	public DealerGroup() {
+		this.dealers = new ArrayList<Dealership>();
+	}
+	
+	public Dealership getDealerByID(String dealerID) {
+		Dealership getDealer = null;
+		for(int i = 0; i > dealers.size(); i++) {
+			if(dealerID.equalsIgnoreCase(dealers.get(i).getDealerID()) ) {
+			getDealer = dealers.get(i);
+			}
+		}
+		return getDealer;
+	}
+	
+	public Dealership getDealerByIndex(int n) {
+		Dealership getDealer = null;
+		for(int i = 0; i > dealers.size(); i++) {
+			if(n == i) {
+				getDealer = dealers.get(i);
+			}
+		}
+		return getDealer;
+	}
+	
+	public String displayDealerVehicles() {
+		for(Dealership dealer : dealers) {
+			for(Vehicle vehicle : dealer.getVehicleInventory().values()) {
+				System.out.println(vehicle);
+			}
+		}
+		return null;
+	}
+	
+	public String getDealer() {
+		for(Dealership dealer: dealers) {
+			System.out.println(dealer);
+		}
+		return null;
+	}
+	
+	public void addIncomingVehicle(Vehicle newVehicle) {

```

Comment:
Hey the changes look great so far but this method needs to accept a vehicle List and iterate through the List of vehicles, not dealers. 

so the method signature needs to look like:
public void addIncomingVehicles(List<Vehicle> variableName)


For each vehicle
    If dealers does not contain a dealer that matches the dealerShipID that a vehicle has
        A new dealer is created based on the dealershipID provided by the vehicle.
        That dealer should be added to the ArrayList
    Otherwise
        You call the addIncomingVehicle() on the existing dealer.